tracer/src/Datadog.Trace/Debugger/Snapshots/Redaction.cs

```diff
@@ -306,9 +306,10 @@ private static bool IsRemovableChar(char c)
 
         public static void SetConfig(DebuggerSettings settings)
         {
-            foreach (var identifier in settings.RedactedIdentifiers)
+            _redactKeywords.UnionWith(settings.RedactedIdentifiers.Select(x => Normalize(x.Trim())));
+            if (settings.RedactedExcludedIdentifiers.Count > 0)
             {
-                _redactKeywords.Add(Normalize(identifier.Trim()));
+                _redactKeywords.ExceptWith(settings.RedactedExcludedIdentifiers.Select(x => Normalize(x.Trim())));
```

Comment:
Why don't you do the same check for `settings.RedactedIdentifiers`?

Reply:
Fixed in 2b26bc0eaacc9dd189d8a367d725dfda88467d86